content/docs/how-we-work/open-source-contribution/_index.md

```diff
@@ -2,14 +2,17 @@
 title: Open Source Contribution
 weight: 11
 bookCollapseSection: true
-description: Open source contributions involve collaborating on free and open software projects to enhance and create software for anyone to use. It allows developers like us to give back to the community, improve our skills, and make a significant impact on the world of technology.
+description: At Axelerant, our approach to open source contributions is guided by our values. We understand that growth is ongoing, and our work on various projects often reveals opportunities to expand our knowledge. To seize these opportunities, we prioritize continuous learning and improvement. Whether we're actively engaged in projects or participating in community initiatives, we do so with dedication and a sense of responsibility. This not only enhances our expertise but also allows us to give back to the open source community, aligning with our mission to make a positive impact. In this way, we uphold the Axelerant way of contributing and our commitment to excellence.
 ---
 
 # Open Source Contribution
 
+At Axelerant, our approach to open source contributions is guided by our values. We understand that growth is ongoing, and our work on various projects often reveals opportunities to expand our knowledge. To seize these opportunities, we prioritize continuous learning and improvement. Whether we're actively engaged in projects or participating in community initiatives, we do so with dedication and a sense of responsibility. This not only enhances our expertise but also allows us to give back to the open source community, aligning with our mission to make a positive impact. In this way, we uphold the Axelerant way of contributing and our commitment to excellence.
```

Comment:
@Yemaneberhan-Lemma, the way I read this, it appears cyclic to me.

You say "opportunities to expand our knowledge" and then "To seize these opportunities, we prioritize continuous learning and improvement." So, with continuous learning and improvement, you seize the opportunity to expand our knowledge. But isn't our knowledge already expanded with continuous learning?

> Whether we're actively engaged in projects or participating in community initiatives, we do so with dedication and a sense of responsibility.

I think this is relevant more as a principle than just for open-source contributions. This would be fine here but the next sentence does not make sense: "This not only enhances our expertise but also allows us to give back to the open source community." How would that dedication help us to give back? I feel there is a missing step in between.

---

All in all, I think this line can be simplified. Right now, people might interpret it differently or just skim through this paragraph. What is it that you really want to say here? Maybe start there and we can continue tweaking it.

Reply:
made changes based on comments.